portal-api/src/v1/users/routes.js

```diff
@@ -31,6 +31,26 @@ const userService = new UserService();
 const signInLinkService = new SignInLinkService(randomGenerator, hasher, userRepository, userService);
 const signInLinkController = new SignInLinkController(signInLinkService);
 
+// const isAdmin = (req, res, next) => {
```

Comment:
Please ensure no commented code is included in the PR for review.

Reply:
Removed comment